mbuild/formats/lammpsdata.py

```diff
@@ -524,6 +597,7 @@ def _get_bond_types(structure, bonds, sigma_conversion_factor,
                                              tuple(sorted((bond.atom1.type,bond.atom2.type)))
                                              ) for bond in structure.bonds])))
     unique_bond_types = OrderedDict([(y,x+1) for x,y in unique_bond_types.items()])
+    print(unique_bond_types)
```

Comment:
Should this print statement be here?

Reply:
Sorry about the delayed response! You are right - that print statement is unnecessary. I probably forgot to remove it while debugging. 

I have removed the line and will push a new commit soon.